docs/developing/troubleshooting.md

```diff
@@ -110,9 +110,9 @@ ImportError: No such module requests
 **How to fix the problem:** Our recommendations are as follows.
 
 - For Python apps, **add the app's virtualenv to alwaysInclude.** To do that, look for the
-  `alwaysInclude` list and add `opt/app/env`. See the [ContactOtter sandstorm-pkgdef.capnp for an
+  `alwaysInclude` list and add `opt/app-venv`. See the [ContactOtter sandstorm-pkgdef.capnp for an
```

Comment:
I wonder if there's another app we could point to here that's closer to being maintained? Also I see some potential  for confusion here since (I assume) that app pulls in `opt/app/env`.

Reply:
The problem is there are no example apps currently showing that location, since that's a new placement as of today's merge. So I could drop the example entirely, but then I don't have an example for the formatting of that field in the pkgdef, which I feel has value.

I think I will leave it for the moment, but I have a strong desire to revisit this in future passes.